packages/trip-form/src/settings-selector-panel.story.js

```diff
@@ -1,15 +1,20 @@
 import { ClassicModeIcon } from "@opentripplanner/icons";
 
-import React, { Component } from "react";
 import { action } from "@storybook/addon-actions";
+import React, { Component } from "react";
+import styled from "styled-components";
 
 import SettingsSelectorPanel from "./SettingsSelectorPanel";
+import TripOptions from "./TripOptions";
 
 import commonCompanies from "./__mocks__/companies";
 import commonModes from "./__mocks__/modes";
 import commonModesEmpty from "./__mocks__/modes-empty";
 import trimet from "./__mocks__/trimet.styled";
 
+// SVG icons
+import { walkIcon, bikeIcon, carIcon, scooterIcon } from "./__mocks__/icons";
```

Comment:
The method by which the SVGs are imported into the component is far from perfect. However, it looks like (at least when using storybook) importing the files will be even more difficult: https://duncanleung.com/import-svg-storybook-webpack-loader/

At the moment the component accepts any inline svg React. This _should_ mean that when importing an svg file, it can be added to the iconMap object quite easily.